src/go/configmanager/flags/flags.go

```diff
@@ -160,6 +162,7 @@ func EnvoyConfigOptionsFromFlags() options.ConfigGeneratorOptions {
 		SuppressEnvoyHeaders:                    *SuppressEnvoyHeaders,
 		UnderscoresInHeaders:                    *UnderscoresInHeaders,
 		ServiceControlNetworkFailOpen:           *ServiceControlNetworkFailOpen,
+		EnableGrpcForHttp1:                      *EnableGrpcForHttp1,
```

Comment:
Do we need this flag here, or just default to false? I thought this is only needed for ApiGateway when it runs on Titanium.

Reply:
+1.

Reply:
Decided to add a Option to ConfigMgr.  We don't need a flag.  But for now,  every ConfigMgr option has a flag, so just add a flag.